docs/getting-started/install-configure.md

```diff
@@ -5,44 +5,28 @@ weight: 2
 indent: true
 ---
 
-# Choosing Hosted or Self-Hosted Crossplane
+# Choosing Your Crossplane Distribution
 
 Users looking to use Crossplane for the first time have two options available to
-them today. The first way is to use a hosted Crossplane service like [Upbound
-Cloud][Upbound Cloud]. Alternatively, users looking for some more
-flexibility can install Crossplane into their own Kubernetes cluster.
+them today. The first way is to use the upstream distribution of Crossplane 
+which is maintained and released by the community. 
 
-Crossplane will be installed using the regularly published Helm chart. The Helm
-chart contains all the custom resources and controllers needed to deploy and
-configure Crossplane.
+The second option is to use a vendor supported Crossplane distribution. These 
+distributions are fully compatible with upstream Crossplane, but may include 
+additional features or tooling around it that makes it easier to use in 
+production environments. 
```

Comment:
Should we mention something about distributions being conformant? e.g. https://github.com/cncf/crossplane-conformance/pull/16

Reply:
That's a great point. Added here. https://github.com/crossplane/crossplane/pull/2819/commits/270b8608fe7f908e538727f1ee4db605d1594de3